pkg/gardenlet/controller/shoot/shoot_control_migrate.go

```diff
@@ -311,11 +367,16 @@ func (r *shootReconciler) runPrepareShootForMigrationFlow(ctx context.Context, o
 			Fn:           botanist.Shoot.Components.Extensions.Infrastructure.WaitMigrate,
 			Dependencies: flow.NewTaskIDs(migrateInfrastructure),
 		})
-		_ = g.Add(flow.Task{
+		destroyInfrastructure = g.Add(flow.Task{
```

Comment:
```suggestion
		deleteInfrastructure = g.Add(flow.Task{
```
So that it is aligned with other variable names.